tools/setup_RedHat.sh

```diff
@@ -63,14 +15,12 @@ sudo yum -y install libffi-devel openssl-devel docker-ce btrfs-progs
 # Disable SELinux
 setenforce 0
 
-setup_disk
-
 # Setup Docker
 sudo mkdir /etc/systemd/system/docker.service.d
 sudo tee /etc/systemd/system/docker.service.d/kolla.conf << EOF
 [Service]
 ExecStart=
-ExecStart=/usr/bin/dockerd --storage-driver btrfs --insecure-registry=0.0.0.0/0
+ExecStart=/usr/bin/dockerd --storage-driver overlay2 --insecure-registry=0.0.0.0/0
```

Comment:
This looks quite significant!

Reply:
It's just a script for setting up CI nodes.